crates/apub/src/http/routes.rs

```diff
@@ -19,39 +19,43 @@ use actix_web::{
   http::{header, Method},
   web,
 };
+use lemmy_utils::cache_header::cache_1hour;
 
 pub fn config(cfg: &mut web::ServiceConfig) {
-  cfg
-    .route("/", web::get().to(get_apub_site_http))
-    .route("/site_outbox", web::get().to(get_apub_site_outbox))
-    .route(
-      "/c/{community_name}",
-      web::get().to(get_apub_community_http),
-    )
-    .route(
-      "/c/{community_name}/followers",
-      web::get().to(get_apub_community_followers),
-    )
-    .route(
-      "/c/{community_name}/outbox",
-      web::get().to(get_apub_community_outbox),
-    )
-    .route(
-      "/c/{community_name}/featured",
-      web::get().to(get_apub_community_featured),
-    )
-    .route(
-      "/c/{community_name}/moderators",
-      web::get().to(get_apub_community_moderators),
-    )
-    .route("/u/{user_name}", web::get().to(get_apub_person_http))
-    .route(
-      "/u/{user_name}/outbox",
-      web::get().to(get_apub_person_outbox),
-    )
-    .route("/post/{post_id}", web::get().to(get_apub_post))
-    .route("/comment/{comment_id}", web::get().to(get_apub_comment))
-    .route("/activities/{type_}/{id}", web::get().to(get_activity));
+  cfg.service(
+    web::scope("")
+      .wrap(cache_1hour())
+      .route("/", web::get().to(get_apub_site_http))
+      .route("/site_outbox", web::get().to(get_apub_site_outbox))
+      .route(
+        "/c/{community_name}",
+        web::get().to(get_apub_community_http),
+      )
+      .route(
+        "/c/{community_name}/followers",
+        web::get().to(get_apub_community_followers),
+      )
+      .route(
+        "/c/{community_name}/outbox",
+        web::get().to(get_apub_community_outbox),
+      )
+      .route(
+        "/c/{community_name}/featured",
+        web::get().to(get_apub_community_featured),
+      )
+      .route(
+        "/c/{community_name}/moderators",
+        web::get().to(get_apub_community_moderators),
+      )
+      .route("/u/{user_name}", web::get().to(get_apub_person_http))
+      .route(
+        "/u/{user_name}/outbox",
+        web::get().to(get_apub_person_outbox),
+      )
+      .route("/post/{post_id}", web::get().to(get_apub_post))
```

Comment:
Still a little scary to me that all of these things are going to be an hour old.

Reply:
Lemmy only updates them once a day so it doesnt make much difference.